ani-cli

```diff
@@ -337,33 +389,27 @@ while [ $# -gt 0 ]; do
                 *) player_function="syncplay" ;;
             esac
             ;;
-        -q | --quality)
-            [ $# -lt 2 ] && die "missing argument!"
-            quality="$2"
-            shift
-            ;;
         -S | --select-nth)
             [ $# -lt 2 ] && die "missing argument!"
             index="$2"
             shift
             ;;
-        -c | --continue) search=history ;;
-        -d | --download) player_function=download ;;
-        -D | --delete)
-            : >"$histfile"
-            exit 0
+        -t | --thumbnail)
+            dep_ch "ueberzugpp"
+            thumbnails=1
             ;;
-        -V | --version) version_info ;;
-        -h | --help) help_info ;;
-        -e | --episode | -r | --range)
-            [ $# -lt 2 ] && die "missing argument!"
-            ep_no="$2"
-            [ -n "$index" ] && ANI_CLI_NON_INTERACTIVE=1 #Checks for -S presence
-            shift
+        -U | --update) update_script ;;
+        -v | --vlc)
+            case "$(uname -a)" in
+                *ndroid*) player_function="android_vlc" ;;
+                MINGW* | *WSL2*) player_function="vlc.exe" ;;
+                *ish*) player_function="iSH" ;;
+                *) player_function="vlc" ;;
```

Comment:
Since plattform specific patterns arise in several parts of the script, it may be wise to put it in it's own function to run on start.
This can then handly vlc and syncplay paths, among others.
Downside of that approach would be the introduction of new variables and slightly increased complexity due to the indirection.

Reply:
nvm, for another day maybe